MixItUp.Base/Resources.de-DE.resx

```diff
@@ -638,27 +638,27 @@
     <comment>Manual</comment>
   </data>
   <data name="Event" xml:space="preserve">
-    <value>Event</value>
+    <value>Ereigniss</value>
     <comment>Manual</comment>
   </data>
   <data name="EventBonuses" xml:space="preserve">
     <value>Event Bonusse</value>
     <comment>Manual</comment>
   </data>
   <data name="EventList" xml:space="preserve">
-    <value>Eventliste</value>
+    <value>Ereignissliste</value>
```

Comment:
should be "Ereignisliste" not "Ereignis**s**liste"

Reply:
Please do not use pull request comments to recommend changes to strings. The best place is to either make a PR yourself, or come post in our localization channel on our discord.